reaction.json

```diff
@@ -48,5 +48,10 @@
     "name": "balais couille",
     "match": "balai(.)couille|ballek",
     "reactions": ["balai", "couilles"]
+  },
+   {
```

Comment:
fix the linting and also improve your commit messages

Reply:
Linting updated. Sadly working on GitHub.com (didn't pulled locally) so Im currently unable to update previous commit messages